apps/src/music/blockly/blocks/simple2.js

```diff
@@ -210,20 +212,12 @@ export const setEffectAtCurrentLocationSimple2 = {
       {
         type: 'field_dropdown',
         name: FIELD_EFFECTS_NAME,
-        options: [
-          ['volume', 'volume'],
-          ['filter', 'filter'],
-          ['delay', 'delay'],
-        ],
+        options: FIELD_EFFECT_NAME_OPTIONS,
       },
       {
-        type: 'field_dropdown',
+        // This input is replaced with a field_dropdown by the extension
+        type: 'input_dummy',
```

Comment:
Also for my learning, is this a standard practice (my blockly is rusty 😬 )?

Reply:
FWIW, I see it at https://github.com/code-dot-org/code-dot-org/blob/7c1b5bee73ca9ec2992642b3fef2b825f8b5ce33/apps/src/blockly/customBlocks/googleBlockly/behaviorBlocks.ts#L89.

Reply:
Yeah, I do think this is a blockly standard as per these docs as well: https://developers.google.com/blockly/guides/create-custom-blocks/define-blocks#block_inputs

Reply:
Not thrilled about the name, but I got it from this doc: https://developers.google.com/blockly/guides/create-custom-blocks/fields/built-in-fields/dropdown#dynamic_dropdowns

Reply:
Nice, thanks for the info y'all!